evm/db/chain.py

```diff
@@ -40,6 +41,9 @@
 )
 
 
+CANONICAL_HEAD_HASH_DB_KEY = b'canonical_head_hash'
```

Comment:
Thoughts on adding a version prefix to this `b'v1:canonical_head_hash'`.  It seems very likely that we'll be iterating on our database schema and it might be nice to have a version namespace from the start.

Reply:
Sorry, forgot about this from the other PR. Yeah, I'm definitely fine with that